modules/core/test/v2/unit/coins/eosutil/eosTextDecoder.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @prettier
+ */
+
+import { EosTextDecoder } from '../../../../../src/v2/coins/eosutil/eosTextDecoder';
+import 'should';
+
+describe('Eos Text Decoder', function () {
```

Comment:
i think we also need to test if both string & test decoder can encode and decode the same message with the same output each time. Ideally we use a sample EOS transaction for this test.

Reply:
Updated - added a test that encodes a packed_trx hex string.
then decodes it back using both StringTextDecoder and TextDecoder